.travis.yml

```diff
@@ -26,6 +27,12 @@ script:
   - dotnet restore "SampleApplications/Workshop/Reference/UA Reference.sln"
   - dotnet build -c Debug SampleApplications/Workshop/Reference/ConsoleReferenceServer
   - dotnet build -c Release SampleApplications/Workshop/Reference/ConsoleReferenceServer
+  - dotnet build -c Debug SampleApplications/Samples/GDS/ConsoleServer
+  - dotnet build -c Release SampleApplications/Samples/GDS/ConsoleServer
+  - dotnet build -c Debug SampleApplications/Samples/GDS/ClientTest
+  - dotnet build -c Release SampleApplications/Samples/GDS/ClientTest
+  - dotnet test -c Debug SampleApplications/Samples/GDS/ClientTest
+  - dotnet test -c Release SampleApplications/Samples/GDS/ClientTest
```

Comment:
still failing tests in travis

Reply:
fixed the cert update to not require reopening the listener ports

---
In reply to: [154998191](https://github.com/OPCFoundation/UA-.NETStandard/pull/284#discussion_r154998191) [](ancestors = 154998191)